src/firefly/js/tables/ui/BasicTableView.jsx

```diff
@@ -328,20 +331,22 @@ function correctScrollLeftIfNeeded(totalColWidths, scrollLeft, width, triggeredB
     return scrollLeft;
 }
 
-function columnWidthsInPixel(columns, data, minWidth=45) {
+function columnWidthsInPixel(columns, data, minWidth) {
 
     const maxVals = getColMaxValues(columns, data, {maxColWidth: 100, maxAryWidth: 30});
+    return maxVals.map((text, idx) => colWidthInPixel(text, columns[idx]), minWidth);
+}
 
-    const paddings = 8;
-    return maxVals.map((text, idx) => {
-        const header = columns[idx].label || columns[idx].name;
-        const style = header === text ? headerStyle : {fontSize:12};
-        text = text.replace(/[^a-zA-Z0-9]/g, 'O');    // some non-alphanum values can be very narrow.  use 'O' in place of them.
-        const pxNum =  getPxWidth({text, ...style}) + paddings;
-        return pxNum < minWidth ? minWidth : pxNum;
-    });
+const paddings = 8;
```

Comment:
is this used by anything other than `colWidthInPixel()`?

if not, it might be worth defining it inside the function or (even better) making it a parameter